ldi-core/version-object-creator/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/VersionObjectCreator.java

```diff
@@ -47,8 +47,11 @@ private MemberInfo extractMemberInfo(Model linkedDataModel, Resource memberTypeR
 			Property dateObservedProperty) {
 		String dateObserved = Optional.ofNullable(dateObservedProperty)
```

Comment:
Why do with start with wrapping this in an observable? Can't we remove this first part and just start with the model.listStatements...